adr/ADR-8.md

```diff
@@ -180,11 +191,12 @@ A bucket is a Stream with these properties:
  * The main write bucket must be called `KV_<Bucket Name>`
  * The ingest subjects must be `$KV.<Bucket Name>.>`
  * The bucket history is achieved by setting `max_msgs_per_subject` to the desired history level
+ * Safe key purges that deletes history requires rollup to be enabled for the stream using `rollup_hdrs`
  * Write replicas are File backed and can have a varying R value
  * Key TTL is managed using the `max_age` key
  * Duplicate window must be same as `max_age` when `max_age` is less than 2 minutes
  * Maximum value sizes can be capped using `max_msg_size`
- * Maximum number of keys cannot currently be limited - we will add something to the server
+ * Maximum number of keys cannot currently be limited
```

Comment:
We should post an issues against server, this one feels like would be used alot.